### ISSUE TITLE
Profile picture not resizing to fit card's size

### ISSUE THREAD

**Issue by ghost** [codurance/site#1965]
In the [Our People](https://codurance.com/about-us/our-people/) page, the pictures are not being resized to the size of the card, causing a visual inconsistency.

![image](https://user-images.githubusercontent.com/57622772/90775495-d44bfe80-e2f0-11ea-8791-275e0c9ee8d3.png)

The image should be resized to fit the card's size.

**Comment by mhughes4** [comment]
Yes I have noticed this with quite a few, including the image for Lesmes Lopez.
